src/status_im/contexts/profile/backup_recovery_phrase/view.cljs

```diff
@@ -108,8 +109,7 @@
             [quo/bottom-actions
              {:actions          :one-action
               :button-one-label (i18n/label :t/i-have-written)
-              :button-one-props {:disabled?           (some false? (vals @checked?))
-                                 :customization-color customization-color
```

Comment:
In the normal flow, `revealed?` can only be toggled once all checks are complete. Therefore, the `:disabled?` check is unnecessary.
For the `revealed? true` flow (see recovery phrase again), we don't care about checks.